util/errorNotifier/bugsnag.js

```diff
@@ -1,5 +1,9 @@
 /* eslint-disable no-param-reassign */
 const Bugsnag = require("@bugsnag/js");
+const {
+  CustomError: CDKCustomError,
```

Comment:
Where do we have DataValidationError in rudder-transformer ? Or is this something different ?

Reply:
These are from CDK v1.